types/command-line-args/index.d.ts

```diff
@@ -1,87 +1,86 @@
-// Type definitions for command-line-args 4.0.7
+// Type definitions for command-line-args 5.0.2
```

Comment:
since this is a major version upgrade, mind forking the folder into v4, and current?

see https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version